sycl/include/CL/sycl/detail/helpers.hpp

```diff
@@ -272,5 +244,36 @@ getSPIRVMemorySemanticsMask(const access::fence_space AccessSpace,
 }
 
 } // namespace detail
+
+// sycl::bit_cast ( no longer sycl::detail::bit_cast )
```

Comment:
Probably the definition to be moved from `sycl/include/CL/sycl/detail/helpers.hpp` to dedicated `sycl/include/CL/sycl/bit_cast.hpp` header.

Reply:
thanks.  Moved it.